randovania/data/json_data/prime1.txt

```diff
@@ -999,141 +1644,498 @@ Burning Trail
 Asset id: 1833127758
 > Dock to Lake Tunnel; Heals? False
   * Normal Door to Lake Tunnel/Dock to Burning Trail
+  > Dock to Save Station Magmoor A
+      Trivial
 
 > Dock to Transport to Chozo Ruins North; Heals? False
   * Normal Door to Transport to Chozo Ruins North/Dock to Burning Trail
+  > Dock to Save Station Magmoor A
+      Trivial
 
 > Dock to Save Station Magmoor A; Heals? False
-  * Normal Door to Save Station Magmoor A/Dock to Burning Trail
+  * Missile Blast Shield to Save Station Magmoor A/Dock to Burning Trail
+  > Dock to Lake Tunnel
+      Trivial
+  > Dock to Transport to Chozo Ruins North
+      Trivial
 
 ----------------
 Lake Tunnel
 Asset id: 2037927229
 > Dock to Lava Lake; Heals? False
   * Normal Door to Lava Lake/Dock to Lake Tunnel
+  > Dock to Burning Trail
+      Any of the following:
+          Heated Rooms Damage ≥ 100
+          Space Jump Boots and Heated Rooms Damage ≥ 91
 
 > Dock to Burning Trail; Heals? False
   * Normal Door to Burning Trail/Dock to Lake Tunnel
+  > Dock to Lava Lake
+      Any of the following:
+          Heated Rooms Damage ≥ 87
+          Space Jump Boots and Heated Rooms Damage ≥ 78
 
 ----------------
 Save Station Magmoor A
 Asset id: 162783260
 > Dock to Burning Trail; Heals? False
-  * Normal Door to Burning Trail/Dock to Save Station Magmoor A
+  * Missile Blast Shield to Burning Trail/Dock to Save Station Magmoor A
+  > Save Station
+      Trivial
+
+> Save Station; Heals? True
+  > Dock to Burning Trail
+      Trivial
 
 ----------------
 Lava Lake
 Asset id: 2758909034
 > Dock to Lake Tunnel; Heals? False
   * Normal Door to Lake Tunnel/Dock to Lava Lake
+  > Dock to Pit Tunnel
+      All of the following:
+          Morph Ball
+          Any of the following:
+              All of the following:
+                  Morph Ball Bomb
+                  Any of the following:
+                      Heated Rooms Damage ≥ 514
+                      Space Jump Boots and Heated Rooms Damage ≥ 458
+              All of the following:
+                  Power Bomb ≥ 2
+                  Any of the following:
+                      Heated Rooms Damage ≥ 586
+                      Space Jump Boots and Heated Rooms Damage ≥ 539
+  > Pickup (Artifact of Nature)
+      Any of the following:
+          Missile ≥ 2 and Grapple Beam and Heat-Resisting Suit
+          Missile ≥ 2 and Space Jump Boots and Heated Rooms Damage ≥ 161
+          Space Jump Boots and Heated Rooms Damage ≥ 145 and Shoot Super Missile
```

Comment:
This 3rd path seems irrelevant to me, but if keeping it, add Missile > 5 to it]

Reply:
Shoot Super Missile is a template, it should include the missile ammo requirement.